Solutions/CohesitySecurity/Playbooks/Incident_Email_Playbook/readme.md

```diff
@@ -0,0 +1,22 @@
+# Incident Email Playbook
+## Summary
+This playbook sends emails with all incident details to everyone who needs to participate in the investigation.
+
+## Prerequisites
+1. Cohesty SIEM/SOAR integration needs to be deployed prior to the deployment of this playbook under the same subscription.
```

Comment:
@yinghuang123, please fix the typo Cohesty to Cohesity, thanks.

Reply:
fixed